crates/sozo/src/args.rs

```diff
@@ -38,6 +38,11 @@ pub struct SozoArgs {
     #[command(flatten)]
     pub verbose: clap_verbosity_flag::Verbosity,
 
+    #[arg(long)]
+    #[arg(env = "SOZO_OFFLINE")]
+    #[arg(help = "Run without accessing the network.")]
```

Comment:
```suggestion
    #[arg(hide_short_help = true)]
    #[arg(help = "Run without accessing the network.")]
```

I'd prefer if we hide the CLI short help message for this flag, following `scarb`.